recipes/presidio-analyzer/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "presidio-analyzer" %}
+{% set version = "2.2.355" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/microsoft/presidio/archive/refs/tags/{{ version }}.tar.gz
+  sha256: fc0b6b8ff84da03c79702310e7af060116074ce1a39f6c29433e348eee76707b
+
+build:
+  noarch: python
+  script: cd {{ name }} && {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - poetry-core
+  run:
+    - python >={{ python_min }}, <4
```

Somehow this seems to work but it will fail in the feedstock. We can't have spaces here.

```suggestion
    - python >={{ python_min }},<4
```